server/middlewares/subdomains.coffee

```diff
@@ -0,0 +1,24 @@
+cozyInstance = require '../models/instance'
+application = require '../models/application'
+#for the moment this is a poc !
```

Comment:
Please add a comment to describe what does the middleware. Also remove that one because we won't merge a PoC :p